### PR TITLE
Wants to merge Development Branch into Main branch

### PR DESCRIPTION
**Pull request by eri8-9**
- Complete the work on individual interfaces.
- Record a video for your project.
- Ask for the external review - using your Student Dashboard.

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -0,0 +1,18 @@
+import './style.css';
```
Comment (carlos-ssh) [inline src/index.js:1]: - Kindly work on the requested [design](https://github.com/microverseinc/curriculum-javascript/blob/main/group-capstone/js_capstone.md).
Screen Shot 2021-08-13 at 21 03 02

**Review by carlos-ssh — CHANGES_REQUESTED** [review]
Hello Team, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!



## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._




Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have only 3 code reviews per project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/index.html [hunk 2]
```diff
@@ -0,0 +1,24 @@
+<!DOCTYPE html>
```
[the remark this answers is not in the document]
Comment (eri8-9) [reply src/index.html:1]: - The branch we are using for development is the default branch (development). Our main branch is where we want to merge after receiving approval.

File: src/__test__/counterComments.test.js [hunk 3]
```diff
@@ -0,0 +1,27 @@
+/**
```
[the remark this answers is not in the document]
Comment (eri8-9) [reply src/__test__/counterComments.test.js:1]: - The requirements just ask counters for 2 tests for likes and comments.

File: src/index.html (hunk 2 above)
[the remark this answers is not in the document]
Comment (carlos-ssh) [reply src/index.html:1]: When I reviewed, your main branch was where you are developing. It's nice that it's fixed. :)

File: src/__test__/counterComments.test.js (hunk 3 above)
[the remark this answers is not in the document]
Comment (carlos-ssh) [reply src/__test__/counterComments.test.js:1]: Ok

**Comment by Jmagero** [comment]
-Removed comments
-Add READme
-Add video
-Worked on the design

**Review by hemant-soni-vst-au4 — APPROVED** [review]
Hi @eri8-9 and @Jmagero, 

Good work on making the requested changes. 👏 
Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 

## Congratulations! 🎉 

Alt Text

Cheers and happy coding 😃

Feel free to leave any questions or comments in the PR thread if something is not 100% clear. You can mention me (@hemant-soni-vst-au4) so I can see your comments and respond.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**